Changes

```diff
@@ -1,3 +1,167 @@
+    succession (#2337).
+- File Menu : Fixed "Import..." bugs (#1077, #2339).
+- Edit Menu : Fixed automatic layout of pasted nodes (#2383)
+- ShaderPlug : Fixed bug when deserialising inputs from BoxIO nodes (#2374)
+- Crop : Fixed bug whereby changing the format plug value did not trigger
+  recomputation (#2368).
+- ImageWriter (#2364) :
+  - Fixed a crash when writing a data window with a single pixel .
+  - Fixed writing of empty data windows to write a single pixel data
+    window instead of a full image. OpenEXR does not support empty data
+    windows, so we must write at least one pixel.
+- NodeGraph :
+  - Fixed confusing Box navigation behaviour (#2343, #2347).
+  - Fixed overlay text to always be on top (#674, #2304).
+  - Fixed layering so dragged connections are always drawn behind
+    nodes, and highlighted plugs are drawn above all others (#2304).
+- SubGraph : Accounted for BoxIO nodes in `correspondingInput() (#2360).
+  This fixe a node reconnection bug when deleting a Box or Reference, and
+  allows Box and Reference nodes to be drag-inserted onto existing connections.
+- ScriptNode :
+  - Fixed node deletion problems caused by errors in `correspondingInput()`
+    implementations (#2355). This was preventing nodes representing missing Arnold
+    shaders from being deleted.
+  - Fixed crashes caused by syntax errors in code passed to `execute()` method
+    (#2319, #2320). This could cause a crash when pasting invalid text into the
+    NodeGraph.
+- Shader/ShaderPlug : Fixed GIL management bug which could cause a deadlock
+  in the `attributes()` and `attributesHash()` methods (#2354).
+- GraphComponent : Fixed crashes caused by passing `None` to the Python
+  bindings (#2338).
+- VectorWarp : Fixed handling of nan/inf (#2341).
+- Expression : Fixed copy/paste bug (#2336).
+- BoxIO : Fixed undo/redo of `insert()` method (#2314).
+- RendererAlgo : Fixed crash when renderer returns a null ObjectInterface (#2302).
+- SetUI : Fixed set name context menus to account for intermediate nodes between
+  destination and filter (#2312).
+- StandardLightVisualiser : Fixed for Cortex 10 UV conventions (#2311).
+- PlugAlgo : Fixed promotion of non-serialisable output plugs (#2314).
+- Serialiser : Fixed syntax error created by `moduleDependencies()` (#2320).
+- Plug : Fixed `setInput( nullptr )` so it removes inputs from all descendant
+  plugs (#2323).
+
+API
+---
+
+- OpenColorIOTransform : Added method to query available role names (#2379)
+- Added default template arguments to simplify usage of several methods (#2361)
+  - `PlugGadget::getPlug()`
+  - `IndividualContainer::getChild()`
+  - `Dot::inPlug()` and `Dot::outPlug()`
+  - `View::inPlug()`, `View::getPreprocessor()` and `View::preprocessedInPlug()`
+  - `Metadata::value()`
+  - `BoxIO::plug()` and `BoxIO::promotedPlug()`
+- Added GafferUI.ButtonPlugValueWidget (#2348).
+- ScriptNode : Added `importFile()` method (#2339).
+- Added new OpenGL renderer backend (#2302).
```

Comment:
does this belong under Features?

Reply:
I'm never sure what to do when something falls into more than one category, but I'm trying to make the release notes friendly enough that folks might actually read 'em, which is why I'm tending to only mention the API stuff in the API section. From the user friendliness perspective I'm also disliking the convention of having the breaking changes up top. They're usually relevant only to developers, they sound kinda scary, and they draw attention away from the shiny new features. I'd rather move the breaking changes to the bottom, and have a policy of using the top paragraph to draw attention to anything we think an end user should care about (this time, the imath/IECoreScene changes).

Reply:
> only mention the API stuff in the API section.

Yeah, I like that approach. I was more asking, is the new OpenGL backend an API-only thing, or does it have user facing implications that we might like to publicize (e.g. Improved Viewport rendering)? Maybe in this case there was no such performance improvement to market and it really is just API?

> From the user friendliness perspective I'm also disliking the convention of having the breaking changes up top.

100% agree on that. I didn't mention it here because I know its our convention, but my god you have to read sooo far down to get to a feature...

>  I'd rather move the breaking changes to the bottom, and have a policy of using the top paragraph to draw attention to anything we think an end user should care about (this time, the imath/IECoreScene changes).

Lets do it!

Reply:
> I was more asking, is the new OpenGL backend an API-only thing, or does it have user facing implications that we might like to publicize (e.g. Improved Viewport rendering)?

It's pretty much API-only. The OpenGLRender node uses that now instead of the old backend, but that shouldn't matter to the user.

>> I'd rather move the breaking changes to the bottom...
> Lets do it!

Done. Also moved the rest of the links to the end...